src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public void setColor(Color newColor) {
```

Comment:
not a mistake, but would be a good idea to create a constructor for those two fields